docs/serving/samples/traffic-splitting/README.md

```diff
@@ -94,71 +102,47 @@ kubectl get ksvc stock-service-example --output yaml
 3. The readiness of the Service can be verified through the Service Conditions.
    When the Service conditions report it is ready again, you can access the new
    Revision using the same method as found in the
-   [previous sample](../rest-api-go/README.md#access-the-service) by prefixing
-   the Service hostname with `latest.`.
+   [previous sample](../rest-api-go/README.md#access-the-service) using
+   the Service hostname found above.
 
 ```shell
-curl --header "Host:latest.${SERVICE_HOSTNAME}" http://${INGRESS_IP}
+curl --header "Host: ${LATEST HOSTNAME}" http://${INGRESS_IP}
```

Comment:
This is not a real bash variable and is not defined above.

Reply:
I added a line that extracts the hostname.